### DISCUSSION TITLE
Can object storage sync to other cluster?

### DISCUSSION THREAD

**Discussion by creeew** [seaweedfs/seaweedfs#3151] [Q&A] [answered]
Filer provide filer.sync to sync file storage to other cluster. And object storage have any feature like filer.sync?
Can we use the same fid to get the same file in other cluster after object storage complete the object sync?

**Comment by chrislusf** [comment, marked as answer]
Object storage is built on top of filer. So the same filer.sync works.

I think you are thinking about blob storage with just master and volume servers. Sync is not available for it.

**Reply by creeew** [reply]
My bad, blob storage is what I mean. 

Do you have any dev plan about cross cluster synchronization in blob storage?

**Reply by creeew** [reply]
Different cluster have different fileId in volume. And they don't same volume server and storage in differnet cluster.  It's really hard to sync blob storage between clusters.
